recorder.js

```diff
@@ -23,39 +40,38 @@ function parse_game_stats(){
                     has_nonzero_score = true;
                 }
                 if(score != ""){
+                    if (is_team_game) {
+                        teams[i].push(name);
+                        name = '[' + teams[i][0] + ']' + name;
+                    } else {
+                        /* if not team game check for winner */
+                        if(j == 1){
+                            game['winner'] = name;
+                            winning_score = score;
+                        } else if(j > 1 && score == winning_score){
+                            game['winner'] = game['winner'] + " - TIE - " + name;
+                        }
+                    };
                     game['scores'].push({name, score, kills, deaths});
-                    teams[i].push(name);
-                    if(j == 1){
-                        game['winner'] = name;
-                        winning_score = score;
-                    }
-                    if(j > 1 && score == winning_score){
-                        game['winner'] = game['winner'] + " - TIE - " + name;
-                    }
                 }
             }
         });
     });
-    if (team_game) {
-        let team_one_score = document.querySelector('[data-hook=team1-score]').textContent;
-        let team_one_name = teams[0].reduce(
-            function(prev, curr){return prev + ' ' + curr},
-            'Team: '
-        );
-        let team_two_name = teams[1].reduce(
-            function(prev, curr){return prev + ' ' + curr},
-            'Team: '
-        );
-        let team_two_score = document.querySelector('[data-hook=team2-score]').textContent;
-        if (team_one_score > team_two_score){
-            game['winner'] = team_one_name;  
-        };
-        if (team_one_score < team_two_score) {
-            game['winner'] = team_two_name;
-        };
-        if (team_one_score == team_two_score) {
-            game['winner'] = team_one_name + " - TIE - " + team_two_score;
+    /* In team games, determine winner by highest team score */
+    if (is_team_game) {
+        let winning_team;
+        if (team_scores[0] > team_scores[1]){
+            winning_team = teams[0];
+        } else if (team_scores[0] < team_scores[1]) {
+            winning_team = teams[1];
+        } else if (team_scores[0] == team_scores[1]) {
+            winning_team = teams[0].concat(["- TIE -"], teams[1]);
         };
+        /* Team name formatting */
+        game["winner"] = winning_team.reduce(
+                function(prev, curr){return prev + ' ' + curr},
+                /* no initial state but here's where it would go */
```

Comment:
nit: you can remove this comment. if `reduce` has one argument the reader should assume the first item in the list serves as the initial state.